easybuild/easyconfigs/c/cuDNN/cuDNN-8.4.1.50-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,39 @@
+name = 'cuDNN'
+version = '8.4.1.50'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is
+a GPU-accelerated library of primitives for deep neural networks."""
+
+toolchain = SYSTEM
+
+# note: cuDNN 8.4.1 is not specific to CUDA 11.6,
+# see also https://docs.nvidia.com/deeplearning/cudnn/support-matrix/index.html#cudnn-cuda-hardware-versions
+source_urls = ['https://developer.download.nvidia.com/compute/redist/cudnn/v%(version)s/local_installers/11.6/']
```

Comment:
can't use the `version` template here, or else the download fails (it's `/v8.4.1/` not `/v8.4.1.50/`

Reply:
fixed in 3c81c78f4b